src/shims/unix/linux/epoll.rs

```diff
@@ -61,8 +61,8 @@ pub struct EpollEventInterest {
     data: u64,
     /// Ready list of the epoll instance under which this EpollEventInterest is registered.
     ready_list: Rc<RefCell<BTreeMap<(FdId, i32), EpollEventInstance>>>,
-    /// The file descriptor value that this EpollEventInterest is registered under.
-    epfd_num: i32,
+    /// The epoll file description that this EpollEventInterest is registered under.
```

Comment:
As discussed in the issue, please also add a comment up at the `fd_num` field justifying why we are using a file descriptor number there, not a proper reference.

Reply:
Ah I did this but probably lost it when tidying up the commits with git rebase. It's back now.